cdn/signUrl/SignUrlSample.Tests/SignUrlTests.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 Google Inc. All rights reserved.
```

Comment:
Copy and paste the exact text we use in all our other copyright statements.

Reply:
Done